Swift.gitignore

```diff
@@ -32,6 +36,12 @@ xcuserdata/
 timeline.xctimeline
 playground.xcworkspace
 
+## Finder
+.DS_Store
+
+## Git
+*.orig # Files created by mergetool
```

Comment:
These rules are not specific to Swift, and are covered by `Global/Patch.gitignore`. Please exclude it from this PR.